### PR TITLE
Hossein deprecation link #828

### LINKED ISSUE #828: Deprecation links
These will need to be displayed on the record pages; they can be created using the record relations editing form.
Here's a ticket which discusses some aspects of their display:

https://github.com/ISA-tools/biosharing/issues/1328#issuecomment-809602603

Both recordAssociations and reverseRecordAssociations will need to be checked for "deprecates" relations.

### PR DESCRIPTION
**Pull request by hoseinmirian**
Other record tab added in the InCollection 
other related records tab added in relatedContent
tests remained covered.

### CODE REVIEW

File: src/components/Records/Record/RelatedContent.vue [hunk 1]
```diff
@@ -135,6 +135,7 @@ export default {
         tabs: {
           related_standards: {relation: ['deprecates'], registry: "Standard", data: []},
           related_databases: {relation: ['deprecates'], registry: "Database", data: []},
```
Comment (knirirr) [inline src/components/Records/Record/RelatedContent.vue]:
```suggestion
          related_databases: {relation: [], registry: "Database", data: []},
```

File: src/components/Records/Record/RelatedContent.vue [hunk 2]
```diff
@@ -135,6 +135,7 @@ export default {
         tabs: {
           related_standards: {relation: ['deprecates'], registry: "Standard", data: []},
```
Comment (knirirr) [inline src/components/Records/Record/RelatedContent.vue]:
```suggestion
          related_standards: {relation: [], registry: "Standard", data: []},
```

**Review by knirirr — CHANGES_REQUESTED** [review]
Currently, this branch doesn't show records deprecated by the record being viewed. The suggested changes _should_ fix that; please look at the Genbank record to be certain; this should have one deprecation association and one deprecation reverse association, for testing purposes.

**Comment by hoseinmirian** [comment]
@knirirr thank you. its fixed. now it shows their reverse relationship.

**Review by knirirr — APPROVED** [review]

**Pull request merged**